client/QuickStart/Program.cs

```diff
@@ -0,0 +1,767 @@
+
+            return new KustoConnectionStringBuilder(clusterUrl)
+                .WithAadUserManagedIdentity(clientId);
+        }
+
+
+        /// <summary>
+        ///  Generates Kusto Connection String based on 'AppCertificate' Authentication Mode.
+        /// </summary>
+        /// <param name="clusterUrl">Url of cluster to connect to</param>
+        /// <param name="certificatePath">Given certificate path</param>
+        /// <param name="certificatePassword">Given certificate password</param>
+        /// <param name="applicationId">Given application id</param>
+        /// <param name="tenantId">Given tenant id</param>
+        /// <returns>AppCertificate Kusto Connection String</returns>
+        private static KustoConnectionStringBuilder CreateAppCertificateConnectionString(string clusterUrl,
+            string certificatePath, string certificatePassword, string applicationId, string tenantId)
+        {
+            X509Certificate2 certificate = null;
+            if (certificatePath != null)
+                certificate = new X509Certificate2(certificatePath, certificatePassword);
+            else
+                ErrorHandler("Missing Certificate path!");
+
+            return new KustoConnectionStringBuilder(clusterUrl).WithAadApplicationCertificateAuthentication(
+                applicationId, certificate, tenantId, sendX5c: true);
+        }
+
+
+        /// <summary>
+        /// Basic Table Commands - including AlterMerge Existing Table and Create Table control commands, and
+        /// Existing Number Of Rows query.
+        /// </summary>
+        /// <param name="config">ConfigJson object</param>
+        /// <param name="adminClient">Privileged client to run Control Commands</param>
+        /// <param name="queryProvider">Client to run queries</param>
+        private static void BasicTableCommands(ConfigJson config, ICslAdminProvider adminClient,
+            ICslQueryProvider queryProvider)
+        {
+            if (config.UseExistingTable == true)
+            {
+                if (config.AlterTable == true)
+                    // Tip: Usually table was originally created with a schema appropriate for the data being ingested,
+                    // so this wouldn't be needed.
+                    // Learn More: For more information about altering table schemas, see:
+                    // https://docs.microsoft.com/azure/data-explorer/kusto/management/alter-table-command
+                    AlterMergeExistingTableToProvidedSchema(adminClient, config.DatabaseName,
+                        config.TableName, config.TableSchema);
+
+                if (config.QueryData == true)
+                    // Learn More: For more information about Kusto Query Language (KQL), see:
+                    // https://docs.microsoft.com/azure/data-explorer/write-queries
+                    QueryExistingNumberOfRows(queryProvider, config.DatabaseName, config.TableName);
+            }
+            else
```

Comment:
Space after comment. (Press ctrl+shift+l in rider to reformat the file)

Reply:
just to make sure i understood - put a space line between the any comment and the code? (and this is after reformatting, i am obsessed with it)

Reply:
I mean instead of:
//Tip:
Do
// Tip: